test.js

```diff
@@ -65,8 +65,8 @@ t.test('getMoonIllumination returns fraction and angle of moon\'s illuminated li
 t.test('getMoonTimes returns moon rise and set times', function (t) {
     var moonTimes = SunCalc.getMoonTimes(new Date('2013-03-04UTC'), lat, lng, true);
 
-    t.equal(moonTimes.rise.toUTCString(), 'Mon, 04 Mar 2013 23:57:55 GMT');
-    t.equal(moonTimes.set.toUTCString(), 'Mon, 04 Mar 2013 07:28:41 GMT');
+    t.equal(moonTimes.rise.toUTCString(), 'Mon, 04 Mar 2013 23:54:29 GMT');
+    t.equal(moonTimes.set.toUTCString(), 'Mon, 04 Mar 2013 07:47:58 GMT');
```

Comment:
The values with the new fix are closer to http://www.timeanddate.com and https://www.calsky.com. 
But as written in http://aa.quae.nl/en/reken/hemelpositie.html the moon formulas suncalc use can lead to errors of up to 2.5°.

It looks like the formulas of http://lexikon.astronomie.info/java/sunmoon/sunmoon.html would give better results. http://www.mooncalc.org uses this library and the values seem to be better that suncalc.



> Am 08.01.2016 um 15:18 schrieb Vladimir Agafonkin <notifications@github.com>:
> 
> In test.js <https://github.com/mourner/suncalc/pull/64#discussion_r49191680>:
> 
> > @@ -65,8 +65,8 @@ t.test('getMoonIllumination returns fraction and angle of moon\'s illuminated li
> >  t.test('getMoonTimes returns moon rise and set times', function (t) {
> >      var moonTimes = SunCalc.getMoonTimes(new Date('2013-03-04UTC'), lat, lng, true);
> >  
> > -    t.equal(moonTimes.rise.toUTCString(), 'Mon, 04 Mar 2013 23:57:55 GMT');
> > -    t.equal(moonTimes.set.toUTCString(), 'Mon, 04 Mar 2013 07:28:41 GMT');
> > +    t.equal(moonTimes.rise.toUTCString(), 'Mon, 04 Mar 2013 23:54:29 GMT');
> > +    t.equal(moonTimes.set.toUTCString(), 'Mon, 04 Mar 2013 07:47:58 GMT');
> I wonder if the new times match the info on websites like timeanddate.com, can you check?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/mourner/suncalc/pull/64/files#r49191680>.
>